src/index.js

```diff
@@ -31,23 +31,24 @@ const removeEvent = function(elem, type, eventHandle) {
   }
 };
 
+const controlsMap = [
+  { funcName: 'renderTopLeftControls', key: 'TopLeft' },
```

Comment:
Couldn't `controlsMap` remain within the constructor of `Carousel`?  It can still be defined as you have it with `funcName` assigned to strings instead of `func` assigned to `this.props` values, but since it's so specifically needed for `Carousel` I don't think it needs to be a global `const`.